<?xml version="1.0"?>

<launch>
  <node name="bag_to_cmd_node" pkg="bag_to_cmd" type="bag_to_cmd_node" output="screen">
      <param name="bag_path" value="/home/martin/Documents/bagfiles/walking/walking_test_2.bag" />
      <param name="controller_manager_ns" value="/thor_mang" />
      <param name="update_frequency" value="125" />
  </node>
</launch>
